dist/autocomplete.js

```diff
@@ -159,6 +159,7 @@ var AutoComplete = (function () {
             "Content-type": "application/x-www-form-urlencoded"
         },
         Limit: 0,
+        MinChars: 3,
```

Comment:
Prefer default value at 0 to keep (by default) the same comportment of precedent release.